package.json

```diff
@@ -118,6 +125,7 @@
     "eslint-plugin-promise": "~3.3.0",
     "eslint-plugin-react": "^7.5.1",
     "eslint-plugin-standard": "~2.0.1",
+    "fetch-mock": "^7.0.0-alpha.6",
```

Comment:
Is there any specific feature you need from the alpha version? These things have a tendency to break a lot.